underscore.js

```diff
@@ -94,6 +93,30 @@
     return cb(value, context, Infinity);
   };
 
+  // Similar to ES6's rest param (http://ariya.ofilabs.com/2013/03/es6-and-rest-parameter.html)
+  // This accumulates the arguments passed into an array, after a given index.
+  var restArgs = function(func, startIndex) {
+    startIndex = startIndex == null ? func.length - 1 : +startIndex;
```

Comment:
Ok cool. You could still pass `startIndex` to restArgs if you needed the fallback behavior, that seems reasonable.